static/js/settings_bots.js

```diff
@@ -362,7 +362,7 @@ exports.set_up = function () {
     $("#select_service_name").on("change", () => {
         $("#config_inputbox").children().hide();
         const selected_bot = $("#select_service_name :selected").val();
```

Comment:
Ugh, this PR is exposing a lot of ugly warts in our codebase.  I believe this is part of the embedded bots system, which is probably bit-rotten.  I just opened https://github.com/zulip/zulip/issues/17210 to try to get us some eyeballs on this feature.